UofT-DSI /sql - Assignment two

vikrama33 opened the pull request:
<!-- PULL REQUEST TITLE: UofT-DSI | <Module Name> - Assignment <assignment number>-->

## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)

## What did you learn from the changes you have made?

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?

## How were these changes tested?

## A reference to a related issue in your repository (if applicable)

## Checklist
- [ ] I can confirm that my changes are working as intended

vikrama33 committed:
I updated the assignment2.sql

vikrama33 committed:
I updated the Assignment2.md

monzchan reviewed (changes requested):
Section 2 and 3 look perfect! please upload your graph for section 1, and also provide answer regarding prompt 3

vikrama33 committed:
Add files via upload

I have uploaded the graph for section 1, and also provide answer regarding prompt 3

vikrama33 commented:
Hi [monzchan](https://github.com/monzchan),

I Have uploaded the files as per your request.

Thank you,

Regards
Vik

monzchan reviewed (commented):
Instead of a flow chart, the assignment 2 requires you to create a diagram showing several tables (section) and indicate their relationships. It would be something similar to your assignment 1

vikrama33 commented:
[Farmers Market - Relationship diagram.pdf](https://github.com/user-attachments/files/21972048/Farmers.Market.-.Relationship.diagram.pdf)



Hi @monzchan,
 
I have uploaded the revisesed relationship diagram as per your request.

Thank you,

Regards
Vik
